composer.json

```diff
@@ -14,6 +14,12 @@
         "autocompletion",
         "refactor"
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/dantleech/php-language-server-protocol"
+        }
+    ],
```

Comment:
If this PR seems OK, then we can add a `felixfbecker/language-server-protocol` package to packagist?